pom.xml

```diff
@@ -61,6 +68,32 @@
     </pluginRepositories>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>2.3.0</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-input-step</artifactId>
+            <version>2.12-rc390.24ce2a334298</version>
```

Comment:
This is an unsatisfiable dependency and should not have been released. Every Jenkins istance currently logs the message:

```
Apr 03, 2020 12:48:39 AM hudson.model.UpdateSite$Plugin getNeededDependencies
WARNING: Could not find dependency pipeline-input-step of kubesphere-extension
```

Reply:
thanks for reminding.
Because the new features of the pipeline input plugin were not merged, I used this approach for development. Should I go to the jenkins dev group to discuss PR about the event and try to merge it?
This feature is very useful for us. Or is there any other suggested processing method?

Reply:
Depending on incrementals is a useful approach during development, but releases with this are not a good idea. If you rely on an unreleased feature, your change cannot be released.

I would start by commenting on the PR asking it be merged, and why.

Reply:
Thanks for your reply, I will avoid using `incrementals` as a dependency for future releases.